naughty/subdomains.py

```diff
@@ -0,0 +1,341 @@
+"""
+Blacklists of words for user-dictated subdomains.
+
+This list has been taken from the one at
+https://github.com/ckrailo/Banned-Subdomains. At the time of writing, the
+readme puts the list into the public domain:
+
+    Feel free to fork/copy/whatever you want to do with this list. I release
+    any data in this repository into the public domain, with the understanding
+    that I take no responsibility for what is done with it, I provide no
+    warranty or guarantee for anything, etc.
+
+Many thanks to @ckrailo for making this module possible.
+"""
+
+
+main = (
```

Comment:
There will also be the fortune 500 list, and the "adult" list from the repo mentioned.